src/commands/aas/instrument.ts

```diff
@@ -99,11 +99,16 @@ export class InstrumentCommand extends AasCommand {
 
       return 1
     }
-    const isApiKeyValid = await newApiKeyValidator({
-      apiKey: process.env.DD_API_KEY,
-      datadogSite: process.env.DD_SITE ?? DATADOG_SITE_US1,
-    }).validateApiKey()
-    if (!isApiKeyValid) {
+
+    try {
+      const isApiKeyValid = await newApiKeyValidator({
+        apiKey: process.env.DD_API_KEY,
+        datadogSite: process.env.DD_SITE ?? DATADOG_SITE_US1,
+      }).validateApiKey()
+      if (!isApiKeyValid) {
+        throw Error()
+      }
+    } catch (e) {
```

Comment:
Can we make `maskString()` show `<unset>` for undefined and `<empty>` for an empty string?

Currently the absence of text after the colon looks like a bug imo